layers/+lang/javascript/README.org

```diff
@@ -149,3 +159,14 @@ You can check more [[https://github.com/mooz/js-doc/][here]]
 | Key Binding | Description                        |
 |-------------+------------------------------------|
 | ~SPC m h p~ | Get the path of the value at point |
+
+** REPL (skewer-mode)
+
+| Key Binding | Description                                                             |
+|-------------+-------------------------------------------------------------------------|
+| ~SPC m s i~ | prepares the skewer environment - run this before running REPL commands |
+| ~SPC m s r~ | starts/goes to the skewer REPL                                          |
+| ~SPC m s b~ | loads the current file in the skewer REPL                               |
+| ~SPC m s e~ | evaluates the last expression                                           |
```

Comment:
Does it print anything in the REPL ?
If not then consider using `SPC m e e` instead.